addons/binding/org.openhab.binding.chamberlainmyq/src/main/java/org/openhab/binding/chamberlainmyq/internal/json/Device.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.chamberlainmyq.internal.json;
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+/**
```

Comment:
Please auto-format (with the openHAB eclipse profile) all the files of your binding.